tests/testthat/test-renderer1-knit-print.R

```diff
@@ -1,168 +1,197 @@
-  ##print("before css selector")
-  tr.list <- div$findChildElements(
-    "css selector", "table.legend tr.label_variable")
-  a <- tr.list[[1]]
-  b <- tr.list[[2]]
-  ##print("before show_hide")
-  show_hide <- div$findChildElement("class name", "show_hide_selector_widgets")
-  ##print("before col_selector_widget")
-  widget <- div$findChildElement("class name", "label_variable_selector_widget")
-  list(a178=a,
-       b934=b,
-       show_hide=show_hide,
-       widget=widget)
-}
-
-plot1top <- get_elements("plot1top")
-plot1bottom <- get_elements("plot1bottom")
-
-# get_circles() returns a list of circles so comparing with list(expected, expected)
-
-test_that("clicking top legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(0, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-test_that("clicking bottom legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 0))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-plot1top$show_hide$clickElement()
-s.div <- plot1top$widget$findChildElement("class name", "selectize-input")
-s.div$clickElement()
-
-test_that("top widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(5, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(0, 10))
-  remDr$sendKeysToActiveElement(list("a", key="enter"))
-  expect_equal(get_circles(), list(5, 10))
-  remDr$sendKeysToActiveElement(list("b", key="enter"))
-  expect_equal(get_circles(), list(10, 10))
-})
-
-plot1bottom$show_hide$clickElement()
-s.div <-
-  plot1bottom$widget$findChildElement("class name", "selectize-input")
-s.div$clickElement()
-
-test_that("bottom widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(10, 5))
-  remDr$sendKeysToActiveElement(list(key="backspace"))
-  expect_equal(get_circles(), list(10, 0))
-  remDr$sendKeysToActiveElement(list("a", key="enter"))
-  expect_equal(get_circles(), list(10, 5))
-  remDr$sendKeysToActiveElement(list("b", key="enter"))
-  expect_equal(get_circles(), list(10, 10))
-})
+acontext("knitting multiple animint plots in a single Rmd")
+print("In test")
```

Comment:
please remove print

Reply:
done

Reply:
you wrote "done" but it still has not been removed, did you forget to push?

Reply:
@tdhock sorry for the confusion, I have pushed it.